.eslintrc.cjs

```diff
@@ -15,9 +15,11 @@ const sharedNoRestrictedImportsConfig = {
   ],
 };
 
+/** @type {import("eslint").Linter.Config} */
 module.exports = {
   // this is the highest config lower ones will automatically extend
   root: true,
+  reportUnusedDisableDirectives: true,
```

Comment:
Should work in editors unlike `--report-unused-disable-directives` package.json scripts